rules/framework/vfs_overlay.bzl

```diff
@@ -214,7 +214,7 @@ def _make_root(vfs_parent, bin_dir_path, build_file_path, target_triple, framewo
     return roots
 
 def _provided_vfs_swift_module_contents(swiftmodules, vfs_prefix, target_triple):
-    swiftmodule_file = _find_top_swiftmodule_file(swiftmodules)
+    swiftmodule_file = _find_swiftmodule_directory(swiftmodules)
```

Comment:
Does this work end to end and also inside of Xcode for you after cleaning derived data with the swiftmodule?

Also if `glob` by default has [`exclude_directories`](src/main/java/com/google/devtools/build/lib/skyframe/GlobDescriptor.java) - I wonder if this is picking directories up from elsewhere ?

Reply:
So it looks like the `swiftmodules` var here is passed in [framework.bzl (here)](https://github.com/bazel-ios/rules_ios/blob/f46c1406a4337c5acf3a8ea76dc89d4eee5271df/rules/framework.bzl#L219-L229). These `framework_files` are gathered [here](https://github.com/bazel-ios/rules_ios/blob/master/rules/framework.bzl#L246)

Reply:
See this comment: https://github.com/bazel-ios/rules_ios/pull/521#discussion_r929851643